lib/xmi/sparx.rb

```diff
@@ -368,18 +374,26 @@ class SparxConnectorModel < Shale::Mapper
     end
 
     class SparxConnectorEndRole < Shale::Mapper
+      attribute :name, Shale::Type::String
       attribute :visibility, Shale::Type::String
+      attribute :targetScope, Shale::Type::String
```

Comment:
Can we use `:target_scope` because Ruby methods are in snake case?

Reply:
Fixed in 82d3ae153a1406dbeb68eecc40532ab1acf5e1b7